### PR TITLE
Thing 2 ready for review

### PR DESCRIPTION
**Pull request by ragamouf**

### CODE REVIEW

File: source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+og:url: 'https://www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2'
+robots: 'index,follow'
+title: 'Thing 2: Issues in research data management - ANDS'
+viewport: |
+    width=device-width, initial-scale=1.0, maximum-scale=1.0,
+    user-scalable=0
+---
+# Thing 2: Issues in research data management
+
+**[Jump to Thing 3&gt;&gt;](thing-3.md)**
```
Comment (NicholaBurton) [inline source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md]: **[&lt;&lt;Jump to Thing 1](thing-1.md) &nbsp; &nbsp; [Jump to Thing 3&gt;&gt;](thing-3.md)**

File: source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+Completion of Thing 2 means you've earned a digital badge!
+
+[Claim your badge](https://credly.com/claim/66836/72E-425C-EA6)
+
+Go to [Thing 3](thing-3.md "thing 3")
+Issues in research data management or to [All Things](index.md)
+
+## Challenge Me
+
+### Digital data in Lab Notebooks**
```
Comment (NicholaBurton) [inline source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md]: Some extra ** have snuck in :)

File: source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+**Consider:** your views on whether the concept of the 5 Vs is useful to
+support better management and reuse of Big Data. If you don't think the
+5 Vs is of value, is there another framework or concept model which
+could be useful for exploring data management for big data?
+
+Do you have a question? Want to share a resource?
+
+Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+Tweet to [@ardc_au](https://twitter.com/ardc_au) using \#23things
```
Comment (NicholaBurton) [inline source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md]: lines 108 and 109 should be bulleted

File: source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md (hunk 2 above)
Comment (NicholaBurton) [inline source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md]: Some extra ** have snuck in :)

File: source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md (hunk 1 above)
Comment (NicholaBurton) [inline source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md]: **[&lt;&lt;Jump to Thing 1](thing-1.md) &nbsp; &nbsp; [Jump to Thing 3&gt;&gt;](thing-3.md)**

File: source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+2\. Then read this article [International team of scientists open sources search for malaria cure](https://opensource.com/life/14/6/international-team-open-sources-search-malaria-cure) about how an international team of scientists and citizen scientists are using open source ELNs to speed up a cure for malaria. You can see their open ELNs [here](http://malaria.ourexperiment.org/). Click on Matthew Todd's ELN to see what is recorded.
+3\. Check out [Harvard Medical School’s](http://malaria.ourexperiment.org/) comparative matrix of ELN platforms
+
+**Consider:** your views on data management issues and possible
+solutions for data that is generated, stored and shared via an open ELN.
+
+Do you have a question? Want to share a resource?
+
+Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+Tweet to [@ardc_au](https://twitter.com/ardc_au) using \#23things
```
Comment (NicholaBurton) [inline source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md]: lines 144 and 145 should be bulleted

File: source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+[LibGuide](http://libguides.usc.edu.au/c.php?g=508408&p=3477838 "USC libguide - Consider the following")
+which provides advice for researchers on how to manage their data.
+
+**Consider:** how just one of the data disasters depicted in the cartoon
+could have been avoided.
+
+Do you have a question? Want to share a resource?
+
+Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+Tweet to [@ardc_au](https://twitter.com/ardc_au) using \#23things
```
Comment (NicholaBurton) [inline source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md]: lines 77 and 78 should be bulleted

**Review by NicholaBurton — CHANGES_REQUESTED** [review]
Looking good, just caught a couple of things!

**Review by tom-h — APPROVED** [review]

**Pull request merged**